tests/functional/adapter/utils/test_utils.py

```diff
@@ -33,6 +37,22 @@ class TestAnyValue(BaseAnyValue):
     pass
 
 
+# Not working because of https://github.com/trinodb/trino/issues/13
+@pytest.mark.skip(reason="No wqy to concat an array with null or empty array")
```

Comment:
This is unfortunate. I have tried multiple workarounds but the Trino parser will always fail on the `concat(null, ARRAY[1,2])` even if you put it behind if, try or case when.

Reply:
a typo in way